### PR TITLE
Remove Clone from TokenStream

### PR DESCRIPTION
**Pull request by maciektr**
**Stack**:
- #1708
- #1701 ⬅
- #1704
- #1700
- #1699

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: plugins/cairo-lang-macro/src/types/token.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@ use std::fmt::Display;
 /// An abstract stream of Cairo tokens.
 ///
 /// This is both input and part of an output of a procedural macro.
-#[derive(Debug, Clone)]
+#[derive(Debug)]
```
Comment (maciektr) [inline plugins/cairo-lang-macro/src/types/token.rs:6]: Just so we can store `Bump` per `TokenStream`  in subsequent work.

File: scarb/src/compiler/plugin/proc_macro/host.rs [hunk 2]
```diff
@@ -308,14 +308,15 @@ impl ProcMacroHostPlugin {
                 return all_none;
             }
         };
+        let original = token_stream.to_string();
```
Comment (wawel37) [inline scarb/src/compiler/plugin/proc_macro/host.rs:311]: nit:
```suggestion
        let token_stream_string = token_stream.to_string();
```

File: scarb/src/compiler/plugin/proc_macro/host.rs [hunk 3]
```diff
@@ -597,10 +598,11 @@ impl ProcMacroHostPlugin {
         token_stream: TokenStream,
         stable_ptr: SyntaxStablePtrId,
     ) -> PluginResult {
+        let original = token_stream.to_string();
```
Comment (wawel37) [inline scarb/src/compiler/plugin/proc_macro/host.rs:601]: nit: same as above

**Review by wawel37 — APPROVED** [review]

**Comment by maciektr** [comment]
Closing pull request: commit has gone away

**Pull request closed without merging**